lib/api/controllers/adminController.js

```diff
@@ -54,6 +58,15 @@ class AdminController {
         return this.kuzzle.internalEngine.bootstrap.all();
       });
 
+    if (refresh === 'wait_for') {
+      return promise.then(() => {
+        delete _locks.resetKuzzleData;
+
+        return { acknowledge: true };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)